pkg/pro/api_client.go

```
@@ -124,54 +128,54 @@ func (c *AtmosProAPIClient) UploadAffectedStacks(dto *dtos.UploadAffectedStacksR
-		return fmt.Errorf(ErrFormatString, ErrFailedToUploadStacks, resp.Status)
+		return fmt.Errorf(cfg.ErrFormatString, ErrFailedToUploadStacks, resp.Status)
 	}
-	c.Logger.Trace(fmt.Sprintf("\nUploaded the affected components and stacks to %s", url))
+	log.Debug(fmt.Sprintf("\nUploaded the affected components and stacks to %s", url))
 
 	return nil
 }
 
-// LockStack locks a specific stack..
+// LockStack locks a specific stack.
 func (c *AtmosProAPIClient) LockStack(dto dtos.LockStackRequest) (dtos.LockStackResponse, error) {
```

golangci-lint (1.64.8)

[error] 156-201 lines are duplicate of `pkg/pro/api_client.go:204-248`

(dupl)